test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/EventSourceTests.cs

```diff
@@ -47,16 +49,33 @@ public EventSourceTests()
 
             // capture list here as other tests executing in parallel may log events
             Assert.NotNull(connectionId);
-            var events = _listener.EventData.Where(e => e != null && GetProperty(e, "connectionId") == connectionId).ToList();
+            Assert.NotNull(requestId);
 
-            var start = Assert.Single(events, e => e.EventName == "ConnectionStart");
-            Assert.All(new[] { "connectionId", "scheme", "remoteEndPoint", "localEndPoint" }, p => Assert.Contains(p, start.PayloadNames));
-            Assert.Equal("http", GetProperty(start, "scheme"));
-            Assert.Equal($"127.0.0.1:{port}", GetProperty(start, "localEndPoint"));
+            var events = _listener.EventData.Where(e => e != null && GetProperty(e, "connectionId") == connectionId).ToList();
 
-            var stop = Assert.Single(events, e => e.EventName == "ConnectionStop");
-            Assert.All(new[] { "connectionId" }, p => Assert.Contains(p, stop.PayloadNames));
-            Assert.Same(KestrelEventSource.Log, stop.EventSource);
+            {
```

Comment:
avoiding copy+paste errors. Because the asserts for each event type looks so similar.